pkg/controller/windowsmachine/nodeconfig/nodeconfig.go

```diff
@@ -281,3 +328,66 @@ func getInstanceIDfromProviderID(providerID string) string {
 	providerTokens := strings.Split(providerID, "/")
 	return providerTokens[len(providerTokens)-1]
 }
+
+// ConfigurePrometheus configures Prometheus monitoring on Windows node
+func ConfigurePrometheus(k8sclientset *kubernetes.Clientset) error {
+	// check if we can get IP address
+	windowsIPList, err := getIPAddress(k8sclientset)
+	if err != nil {
+		return fmt.Errorf("could not get IP address")
+	}
+	if err := updateMetricsEndpoint(windowsIPList, k8sclientset); err != nil {
+		return err
+	}
+
+	return nil
+}
+
+// updateMetricsEndpoint upates the endpoint object with the new list of ip addresses from the Windows nodes.
+// It creates an endpoint object if the object is not found.
+func updateMetricsEndpoint(ipAddresses []v1.EndpointAddress, k8sclientset *kubernetes.Clientset) error {
```

Comment:
This will be addressed as a part of [this](https://github.com/openshift/windows-machine-config-operator/pull/142#discussion_r520943875) implementation